src/util.js

```diff
@@ -84,8 +84,8 @@ export function readJSONFile(filePath, fallbackValue) {
  */
 export function readFileFromCache(filePath, fallbackValue) {
     debug(`read file from cache: ${filePath}`);
-    return (path.extname(filePath) === '.json' ? readJSONFile : readFile)
-        .call(null, path.join(getCacheFolder(), filePath), fallbackValue);
+    return (path.extname(filePath) === '.json' ?
+        readJSONFile : readFile)(path.join(getCacheFolder(), filePath), fallbackValue);
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected an identifier and instead saw ')'.<br>Expected an operator and instead saw '('.